qdcore/qd_animation_set.cpp

```diff
@@ -310,25 +310,25 @@ bool qdAnimationSet::load_animations(const qdNamedObject *res_owner) {
 
 bool qdAnimationSet::free_animations(const qdNamedObject *res_owner) {
 	if (qdGameDispatcher * dp = qdGameDispatcher::get_dispatcher()) {
```

Comment:
As I mentioned, I'd appreciate fixing this formatting when you visit the files: `qdGameDispatcher * dp` -> `qdGameDispatcher *dp`